src/e2e/modules/webchat.test.ts

```diff
@@ -8,12 +8,6 @@ const getMessageCount = async (page: Page): Promise<number> => {
   return (await page.$$('.bpw-chat-bubble')).length
 }
 
-const getLastMessage = async (page: Page): Promise<string> => {
-  const messages = await page.$$('.bpw-chat-bubble >div:last-child')
-  const jsHandle = await messages[0].getProperty('innerText')
-  return jsHandle.jsonValue()
-}
-
```

Comment:
It's not being used right now, but I kept it there for future improvements (not a fan of dead code, but I already developed it before switching to expectMatch). 

Expect match finds the string anywhere in the page, so if we do multiple tests, we may need to check the latest message only